doc/live_migration_no_wal_e.md

```diff
@@ -0,0 +1,166 @@
+
+Assuming you are using Kubernetes for your entire application stack, prepare the appropriate client connection configuration parameters to have them ready for update in Kubernetes, and then to perform an application restart once Stolon is up and running with the migrated database.  You can apply any ConfigMaps or Secrets pertaining to the Stolon connection now, but do not restart any pods until the migration is complete
+
+### Perform the migration
+
+#### Caution
+
+Since this method is meant to go against a live produciton database, a few 'dry' runs are recommended, where you do not shut down any writes from the legacy application, ensuring that you have everything configured properly.  It is recommended to fully reset Stolon after each dry run (taking care to remove the Keepers and PersistentVolumeClaims inbetween runs)
+
+#### Stop writes to legacy database
+
+As mentioned in the description, you will need to ensure there are no writes going to the system before executing this.  Since 'pg_basebackup' can do a binary copy of a running database relatively quickly, there is not much downtime required.
+
+The author of this document used this method to perform a live migration with under under 5 minutes of downtime against a 16GB Postgresql database, during off peak hours.
+
+#### Monitor the migration
+
+Determine which Keeper is the master Keeper, and you can monitor this progress by 'kubectl exec' into the keeper pod and watching the process, the /stolon-data (PGDATA) directory, or disk usage. If you have configured the postgresql Server for logging to STDERR, you should be able to see any issues in the 'kubectl logs' output 
+
+#### Execute the migration
+
+Now let's call stolonctl to configure the Point In Time Recovery with the pg_basebackup command.  It is assumed you have the $STOLON_CLUSTER_NAME, $STOLON_STORE_BACKEND, $STOLON_STORE_ENDPOINTS environment variables set for your Stolon installation
+
+```
+stolonctl --cluster-name=${STOLON_CLUSTER_NAME} --store-backend=${STOLON_STORE_BACKEND} --store-endpoints=${STOLON_STORE_ENDPOINTS} init '{"initMode":"pitr","pitrConfig":{"dataRestoreCommand":"PGPASSWORD=legacydbpassword pg_basebackup -D /stolon-data/postgres/ -P -h legacydbhost -U stolon -x", "archiveRecoverySettings":{"restoreCommand": "echo 1"}}}'
+```
+Here, since we are doing a complete backup and restore of all databasefiles and existing WAL files (-x option), the restoreCommand merely needs to succeed, hence the 'echo 1'.
+
+Now let's patch in desired postgresql.conf parameters using stolonctl update.  In this example, we also tweak the various Stolon timeouts for checking health, and performing failovers in one command:
+
+```
+stolonctl --cluster-name=${STOLON_CLUSTER_NAME} --store-backend=${STOLON_STORE_BACKEND} --store-endpoints=${STOLON_STORE_ENDPOINTS} update --patch '{"failInterval":"10s","requestTimeout":"5s","sleepInterval":"3s","pgParameters":{"max_prepared_transactions":"100","max_conections":"1000"}}
+```
```

Comment:
I'll remove changing the stolon timeouts, since it's outside the scope of this doc.